internal/util/projutil/project_util.go

```diff
@@ -85,6 +87,12 @@ func MustInProjectRoot() {
 // TODO(hasbro17): Change this to check for go.mod
 // "build/Dockerfile" may not be present in all projects
 func CheckProjectRoot() error {
+	// If is the kubebuilder base layout
+	if kbutil.HasProjectFile() {
+		return nil
+	}
```

Comment:
Is this change relevant to this PR? According to @hasbro17's TODO comment, we can have this check for `go.mod`, which would work in both cases.

Reply:
I do not think that we should check for the go.mod. We can use the Project file to know if the project is Ansible, GO and/or Helm as we spoke before and remove all these checks.

Reply:
Good point! And just to be clear, this change is relevant because we use `MustInProjectRoot` and before this change, that would fail in KB projects since it is looking for `./build/Dockefile`?